### PR TITLE
[1] Update node configuration

### PR DESCRIPTION
**Pull request by DannyS03**
- tests guides end-to-end for node and val content
- updates configuration steps

### CODE REVIEW

File: docs/src/guide/maintain/node/build-paloma-core.md [hunk 1]
```diff
@@ -118,36 +51,39 @@ in `~/.paloma/config/config.toml`
...
+## Start the light-client daemon (LCD)
 
 For information about the available Paloma REST API endpoints, see the 
 [Swagger documentation](https://lcd.Paloma.dev/swagger/). To enable the REST 
 API and Swagger, and to start the LCD, complete the following steps:
 
-1. Open `~/.paloma/config/app.toml`.
-2. Locate the `API Configuration` section (`[api]`).
-3. Change `enable = false` to `enable = true`.
+- Open `~/.testnet/paloma-testnet-6/app.toml`.
```
Comment (verabehr) [inline docs/src/guide/maintain/node/build-paloma-core.md:65]: this needs to stay `~/.paloma/config/app.toml

**Review by verabehr — CHANGES_REQUESTED** [review]
small change request

File: docs/src/guide/maintain/node/build-paloma-core.md (hunk 1 above)
Comment (DannyS03) [reply docs/src/guide/maintain/node/build-paloma-core.md]: addressed

**Review by verabehr — APPROVED** [review]
lg

**Pull request merged**